changelogs/fragments/62348-yarn-global_install_fix.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - yarn - handle no version when installing module globally (https://github.com/ansible/ansible/issues/55097)
```

Comment:
Does this only affect installing globally, or also installing locally? From the code change, I would assume that installing a single package locally also didn't work before.

Reply:
This would actually effect both, but mostly people only install by name globally as local installs are done via `package.json`, I'll change it though.